src/App.jsx

```diff
@@ -2,14 +2,16 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
 
   const addOne = () => {
-    // write code here
+    setCount(currentCount => currentCount + 1);
   };
 
   const add100 = () => {
-    // write code here
+    for (let i = 0; i < 100; i += 1) {
+      addOne();
+    }
```

Comment:
no need to load the performance so much. You can just add 100, it is much cheaper for a browser then loop